README.md

```diff
@@ -57,31 +60,61 @@ Help Options:
   -h, --help     Show this help message
 
 Available commands:
-  list   List managed instances. (aliases: ls)
-  run    Run a command on the targeted instances.
-  shell  Start an interactive shell. (aliases: sh)
+  describe  Description a document from ssm.
+  list      List managed instances or documents. (aliases: ls)
+  run       Run a command or document on the targeted instances.
+  shell     Start an interactive shell. (aliases: sh)
 ```
 
-#### List usage
+#### List instances usage
 
 ```bash
-$ ssm-sh list --help
+$ ssm-sh list instances --help
 
 ...
-[list command options]
+[instances command options]
       -f, --filter= Filter the produced list by tag (key=value,..)
       -l, --limit=  Limit the number of instances printed (default: 50)
       -o, --output= Path to a file where the list of instances will be written as JSON.
 ```
 
-#### Run/shell usage
+#### List documents usage
+```bash
+$ ssm-sh list documents --help
+
+...
+[documents command options]
+      -f, --filter= Filter the produced list by property (Name, Owner, DocumentType, PlatformTypes)
+      -l, --limit=  Limit the number of instances printed (default: 50)
+```
+
+#### Run cmd/shell usage
+
+```bash
+$ ssm-sh run cmd --help
```

Comment:
Does it make sense to make `command` as the primary and `cmd` as an alias?
Probably it makes things consistent with existing shell/sh.

Reply:
Yepp makes sense, will update to use `command` instead of `cmd`.